### PR TITLE
[fix] Fixed WifiSession dashboard pie chart 

### PR DESCRIPTION
**Pull request by pandafy**
The WifiSession pie chart on the dashboard was not filtering the
queryset by the related device's organization.

**Blockers**

- [x] https://github.com/openwisp/openwisp-utils/pull/313

Checks:

- [x] I have manually tested the proposed changes
- [ ] I have written new test cases to avoid regressions (if necessary)
- [ ] I have updated the documentation (e.g. README.rst)

### CODE REVIEW

File: requirements-test.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-openwisp-utils[qa] @ https://github.com/openwisp/openwisp-utils/tarball/master
+openwisp-utils[qa] @ https://github.com/openwisp/openwisp-utils/tarball/62c6dfac559e914e8186de4aa80c56b03012ba38
```
Comment (pandafy) [inline requirements-test.txt:1]: TODO: Update this before merging!

File: requirements.txt [hunk 2]
```diff
@@ -4,4 +4,4 @@ django-cache-memoize~=0.1.0
 django-nested-admin~=3.4.0
 netaddr~=0.8
 python-dateutil>=2.7,<3.0
-openwisp-utils[rest] @ https://github.com/openwisp/openwisp-utils/tarball/master
+openwisp-utils[rest] @ https://github.com/openwisp/openwisp-utils/tarball/62c6dfac559e914e8186de4aa80c56b03012ba38
```
Comment (pandafy) [inline requirements.txt:7]: TODO: Update this before merging!

File: requirements-test.txt (hunk 1 above)
Comment (nemesisdesign) [reply requirements-test.txt:1]:
```suggestion
openwisp-utils[qa] @ https://github.com/openwisp/openwisp-utils/tarball/5a8f1dea187586d85c77b46e6e3fd725f557e3c6
```

File: requirements.txt (hunk 2 above)
Comment (nemesisdesign) [inline requirements.txt:7]:
```suggestion
openwisp-utils[rest] @ https://github.com/openwisp/openwisp-utils/tarball/5a8f1dea187586d85c77b46e6e3fd725f557e3c6
```

**Review by nemesisdesign — CHANGES_REQUESTED** [review]

**Review by nemesisdesign — APPROVED** [review]

**Pull request merged**